SPECS/libdrm/libdrm.spec

```diff
@@ -207,125 +199,144 @@ mkdir -p %{buildroot}%{_docdir}/libdrm
 %{_udevrulesdir}/91-drm-modeset.rules
 %endif
 
-
 %files devel
 %dir %{_includedir}/libdrm
 %{_includedir}/libdrm/drm.h
 %{_includedir}/libdrm/drm_fourcc.h
 %{_includedir}/libdrm/drm_mode.h
 %{_includedir}/libdrm/drm_sarea.h
 %{_includedir}/libdrm/*_drm.h
-%{_includedir}/libsync.h
-%{_includedir}/xf86drm.h
-%{_includedir}/xf86drmMode.h
 %{_libdir}/libdrm.so
 %{_libdir}/pkgconfig/libdrm.pc
-
-%if %{with libkms}
-%{_includedir}/libkms/
-%{_libdir}/libkms.so
-%{_libdir}/pkgconfig/libkms.pc
-%endif
-
 %if %{with intel}
 %{_includedir}/libdrm/intel_*.h
 %{_libdir}/libdrm_intel.so
 %{_libdir}/pkgconfig/libdrm_intel.pc
 %endif
-
 %if %{with radeon}
-%{_includedir}/libdrm/radeon_*.h
+%{_includedir}/libdrm/radeon_{bo,cs,surface}*.h
 %{_includedir}/libdrm/r600_pci_ids.h
 %{_libdir}/libdrm_radeon.so
 %{_libdir}/pkgconfig/libdrm_radeon.pc
 %endif
-
 %if %{with amdgpu}
 %{_includedir}/libdrm/amdgpu.h
 %{_libdir}/libdrm_amdgpu.so
 %{_libdir}/pkgconfig/libdrm_amdgpu.pc
 %endif
-
 %if %{with nouveau}
 %{_includedir}/libdrm/nouveau/
 %{_libdir}/libdrm_nouveau.so
 %{_libdir}/pkgconfig/libdrm_nouveau.pc
 %endif
-
 %if %{with omap}
 %{_includedir}/libdrm/omap_*.h
 %{_includedir}/omap/
 %{_libdir}/libdrm_omap.so
 %{_libdir}/pkgconfig/libdrm_omap.pc
 %endif
-
 %if %{with exynos}
 %{_includedir}/libdrm/exynos_*.h
 %{_includedir}/exynos/
 %{_libdir}/libdrm_exynos.so
 %{_libdir}/pkgconfig/libdrm_exynos.pc
 %endif
-
 %if %{with freedreno}
 %{_includedir}/freedreno/
 %{_libdir}/libdrm_freedreno.so
 %{_libdir}/pkgconfig/libdrm_freedreno.pc
 %endif
-
 %if %{with tegra}
 %{_includedir}/libdrm/tegra.h
 %{_libdir}/libdrm_tegra.so
 %{_libdir}/pkgconfig/libdrm_tegra.pc
 %endif
-
 %if %{with vc4}
 %{_includedir}/libdrm/vc4_*.h
 %{_libdir}/pkgconfig/libdrm_vc4.pc
 %endif
-
 %if %{with etnaviv}
 %{_includedir}/libdrm/etnaviv_*.h
 %{_libdir}/libdrm_etnaviv.so
 %{_libdir}/pkgconfig/libdrm_etnaviv.pc
 %endif
-
+%{_includedir}/libsync.h
+%{_includedir}/xf86drm.h
+%{_includedir}/xf86drmMode.h
+%if %{with man_pages}
+%{_mandir}/man3/drm*.3*
+%{_mandir}/man7/drm*.7*
+%endif
 
 %if %{with install_test_programs}
 %files -n drm-utils
+%if %{with amdgpu}
+%{_bindir}/amdgpu_stress
+%endif
 %{_bindir}/drmdevice
+%if %{with etnaviv}
 %exclude %{_bindir}/etnaviv_*
+%endif
+%if %{with exynos}
 %exclude %{_bindir}/exynos_*
-%{_bindir}/kms-steal-crtc
-%{_bindir}/kms-universal-planes
-
-%if %{with libkms}
-%{_bindir}/kmstest
 %endif
-
+%if %{with tegra}
+%exclude %{_bindir}/tegra-*
+%endif
```

Comment:
Similar question about excluding the binary when building with `tegra`.

Reply:
Same as above.

Reply:
For `tegra` it looks like Fedora doesn't explicitly exclude it anywhere. Could you only double-check, that we even need this line and the package builds on ARM64? I've started a [buddy build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=370841&view=results) for you to verify that.

Reply:
Do you mean https://src.fedoraproject.org/rpms/libdrm/blob/rawhide/f/libdrm.spec#_279 ?